regimes/co/tax_identity.go

```diff
@@ -170,10 +170,9 @@ var taxIdentityTypeDefs = []*cbc.KeyDefinition{
 // validateTaxIdentity checks to ensure the NIT code looks okay.
 func validateTaxIdentity(tID *tax.Identity) error {
```

Comment:
Similar to my other comment, it may not be obvious that `validateTaxIdentity` functions aren't only called in the context of the regime they are defined in and so, they shouldn't validate anything that is not universal (like the tax code format). Would it be worth adding a comment at the top of each of these functions?

Reply:
Same applies here, copying the same comments everywhere does scale all that well. A README guide for regimes would probably be useful.